src/Microsoft.DotNet.Configurer/xlf/LocalizableStrings.es.xlf

```diff
@@ -66,7 +66,7 @@ For more information on configuring HTTPS see https://go.microsoft.com/fwlink/?l
         <target state="translated">ASP.NET Core
 ------------
 El certificado de desarrollo HTTPS de ASP.NET Core se ha instalado correctamente.
-Para confiar en el certificado (solo Windows y macOS), instale primero la herramienta dev-certs ejecutando “dotnet install tool dotnet-dev-certs -g --version 2.1.0-preview1-final” y, después, ejecute “dotnet-dev-certs https --trust”.
+Para confiar en el certificado (solo Windows y macOS), instale primero la herramienta dev-certs ejecutando "dotnet install tool dotnet-dev-certs -g --version 2.1.0-preview1-final" y, después, ejecute "dotnet-dev-certs https --trust".
```

Comment:
@v-zbsail I switched to straight quotes because it ends up going through a best-fit-mapping on the console code page of our test infrastructure. I notice that the other translations in this file use straight quotes so this is at least consistent. Can you make the change in TM?

Reply:
Thank you.